README.md

```diff
@@ -31,20 +30,28 @@
 ### Tech Stack <a name="tech-stack"></a>
 
 <details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
-  </ul>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  
 </details>
 
 <details>
-  <summary>Server</summary>
+<summary>Linters</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Node.js</a></li>
+    <li>Lighthouse</li>
+    <li>Webhint</li>
+    <li>Stylelint</li>
   </ul>
 </details>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🚀 Live Demo <a name="live-demo"></a>
 
-<!-- Features -->
+- [Live Demo Link](https://github.com/kifle23/Portfolio-setup-and-mobile-first/)
```

Comment:
[OPTIONAL] The live demo link is not working but only redirects you to your github repository which is not a good user experience and does not make your readme professional enough. Kindly consider removing that part.